pkg/controller/perconaservermongodbbackup/perconaservermongodbbackup_controller.go

```diff
@@ -223,13 +223,7 @@ func (r *ReconcilePerconaServerMongoDBBackup) checkFinalizers(cr *psmdbv1.Percon
 func (r *ReconcilePerconaServerMongoDBBackup) updateStatus(cr *psmdbv1.PerconaServerMongoDBBackup) error {
 	err := r.client.Status().Update(context.TODO(), cr)
```

Comment:
should we use `RetryOnConflict` here too? 

https://github.com/percona/percona-server-mongodb-operator/blob/main/pkg/controller/perconaservermongodb/status.go#L216-L229

Reply:
done